js/jest_get_defaults.js

```diff
@@ -0,0 +1,29 @@
+/* Systematic's jest base config file */
+// see https://facebook.github.io/jest/docs/en/configuration.html
+
+'use strict'
+
+module.exports = function (basePath) {
+  return {
+    moduleFileExtensions: [
+      'js',
+      'json',
+      'vue',
+    ],
+    transform: {
+      '.*\\.(vue)$': `${basePath}/node_modules/vue-jest`,
+      '^.+\\.js$': `${basePath}/node_modules/babel-jest`,
+    },
+    transformIgnorePatterns: [
+      'node_modules/(?!(cassets|vanitils)/)',
+    ],
+    modulePaths: [
+      basePath,
+    ],
+    moduleNameMapper: {
+      '\\.(html|css|less)$': 'identity-obj-proxy',
+    },
+    testRegex: '.spec.js',
```

Comment:
well, we do have a config option to choose the regex already, would be better to report it here or many tests won't be run anymore.